### DISCUSSION TITLE
change the default auto complete in ts when typing <Comp /

### DISCUSSION THREAD

**Discussion by rayBlock** [LazyVim/LazyVim#1828] [Q&A] [answered]
I'm new to neovim / lazy and would like to find out where a behaviour comes from

When I type <Comp / it automatically fills in another "Comp" and closes with a > resulting in <Comp /Comp>

Now I can imagine why you want to automatically have the closing tags for the component but yeah I need to adjust those settings.
 
What would be a good way to find out where this is coming from without going trough disabling all the plugins one by one...

**Comment by dpetka2001** [comment]
I believe that the plugin responsible for closing tags is `windwp/nvim-ts-autotag`. From its [docs](https://github.com/windwp/nvim-ts-autotag#default-values) it seems that you are able to override its default values or you might want to disable it altogether.

**Reply by rayBlock** [reply]
yeah that looks like it - now how would I override that one option. Sorry I'm using this since like 2 days and I barely grasp the basics

**Comment by rayBlock** [comment]
https://github.com/windwp/nvim-ts-autotag/issues/125 should have been fixed but still around apparently

**Reply by dpetka2001** [reply, marked as answer]
You could disable the `enable_close_on_slash` if it would be more convenient for you. Something like 
```lua
return {
  { 
	"windwp/nvim-ts-autotag",
	opts = {
	  enable_close_on_slash = false,
	},
 },
}
```
in a file in your `/lua/plugins/` directory.

**Reply by rayBlock** [reply]
thank you
